### DISCUSSION TITLE
Welcome to ros2-mecanum-bot Discussions!

### DISCUSSION THREAD

**Discussion by deborggraever** [deborggraever/ros2-mecanum-bot#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pr0digy1337** [comment]
Hey there can i implement this on a real hardware robot if please can someone please guide. Im new ROS 2

**Reply by deborggraever** [reply]
Yes, this code repo works on real hardware:

See: [Firmware Repo](https://github.com/deborggraever/ros2-mecanum-bot-firmware)
This firmware is writen for [ATMEGA4809 CURIOSITY NANO](https://www.microchip.com/en-us/development-tool/dm320115)

Driver / motor driver files show the pins used for the motors, can be used with popular L298N motor drivers.

The firmware is experimental / undocumented code, but should be usable.
To implement on other hardware, implementing the HDLC protocol should do fine to work with the ROS hardware node
